src/libraries/System.IO.FileSystem/tests/Directory/EnumerableTests.netcoreapp.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public partial class EnumerableTests : FileSystemTest
+    {
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void EnumerateDirectories_TrailingDot()
+        {
+            string prefix = @"\\?\";
+            string tempPath = GetTestFilePath();
+            string fileName = "Test.txt";
+
+            string[] dirPaths = {
+                 Path.Join(prefix, tempPath, "Test"),
+                 Path.Join(prefix, tempPath, "TestDot."),
+                 Path.Join(prefix, tempPath, "TestDotDot..")
+             };
+
+            // Create directories and their files using "\\?\C:\" paths
+            foreach (string dirPath in dirPaths)
+            {
+                if (Directory.Exists(dirPath))
```

Comment:
In what situation will it exist?

Reply:
I can investigate and add a comment. I moved this unit test from the other file because the enumeration APIs are not available in Framework.

Reply:
The TestDirectory that GetTestFilePath uses is unique, this shouldn't be needed.